p2p/src/authenticated/mod.rs

```diff
@@ -444,7 +445,10 @@ mod tests {
         let cfg = tokio::Config::default();
         let (executor, runtime) = tokio::Executor::init(cfg.clone());
         executor.start(async move {
-            run_network(runtime, cfg.max_message_size, 3000, 10, Mode::One).await;
+            let max_message_size = 1_024 * 1_024; // 1MB
```

Comment:
nit: You can define a `const` inline...usually that is preferred over using `let` here (for static values).